### ISSUE TITLE
Can't open pock

### ISSUE THREAD

**Issue by GeorgeUgly** [pigigaldi/Pock#131]
I downloaded Pock via pock.dev, unzip pock_0_5_2_lack.zip, and double click Pock.app, alert shows "Application Pock can't be opened"

**Desktop (please complete the following information):**
 - MACOS 10.14.5

**Comment by pigigaldi** [comment]
That's strange because I have the same macOS version and I can run it on without issues. 🤔 
Could you please provide more information about your setup?

**Comment by lopsidedclown59** [comment]
Hi I'm having the same issue as George... I'm using a 2018 Macbook pro 13" and I've tried redownloading it multiple times. I've switched accounts and it works on some accounts but won't open on others. I'm also using macOS 10.14.5 and have downloaded V0.5.3 of Pock. Access has been granted to pock in the system preferences and the touch bar has been set to display app controls. Hope this helps
